### ISSUE TITLE
[FEATURE] OAuth 2.0 Device Authorization Grant Enabled

### ISSUE THREAD

**Issue by KirschbaumP** [mrparkers/terraform-provider-keycloak#641]
Keycloak supports **OAuth 2.0 Device Authorization**.

Is it possible to add this property?

```terraform
resource "keycloak_openid_client" "sample_client" {
  realm_id = keycloak_realm.realm.id
  client_id = "sample_client"

  name = "sample_client"
  description = "Sample Client"

  standard_flow_enabled = false
  implicit_flow_enabled = false
  direct_access_grants_enabled = false
  oauth_device_authorization_grant_enabled = true
}
```

**Comment by jitbasemartin** [comment]
Seems already done but not released: https://github.com/mrparkers/terraform-provider-keycloak/pull/578/

**Comment by mrparkers** [comment]
This should be available to use in v3.7.0 now.
